webapp/packages/core-utils/src/copyToClipboard.test.ts

```diff
@@ -0,0 +1,26 @@
+import { copyToClipboard } from './copyToClipboard';
+
+document.execCommand = jest.fn();
```

Comment:
probably you need to reset this after test finished

Reply:
After the test is executed, Jest automatically resets all the mock functions to their original implementations. I moved this part of the code to `BeforeAll()` just in case